### PR TITLE
[Support Us Page] Budget of Each Phase

### PR DESCRIPTION
**Pull request by CarsonKiibi**
**Budget of each phase component**
-simple transition animation
-mobile auto / play button works
-placeholder image
-layout changes from mobile/desktop as specified 
-using "primary" colour from tailwind which is different from figma color but they are both called primary so i assume the tailwind config just needs to be changed to that

biomod-phases1v2

biomod-phases2

### CODE REVIEW

File: src/stories/supportus/PhaseBudget.stories.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import PhaseBudget from "@/components/supportus/PhaseBudget";
+
+const meta = {
+  title: "Support Us / Phase Budget",
```
Comment (daniel-panhead) [inline src/stories/supportus/PhaseBudget.stories.ts]: Very nitpick but the spaces separating the slash aren't needed

**Review by daniel-panhead — CHANGES_REQUESTED** [review]
Very very clean + polished :D

For the transitions I would recommend trying framer motion to make things simpler, although what you have right now works too so it's not required. I would also make the transition a bit shorter as right now it feels just a bit sluggish - maybe drop it to like 150ms? But otherwise this is stellar

File: src/components/supportus/PhaseBudget.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book.",
+  },
+  {
+    image: ideaImg,
+    header: "Phase 4: Validation",
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book.",
+  },
+];
+
+const PhaseBudget: React.FC = () => {
```
Comment (IcedTea2K) [inline src/components/supportus/PhaseBudget.tsx]: Nothing big, just for consistency (and preferences)
```suggestion
function PhaseBudget() {
```

**Review by IcedTea2K — CHANGES_REQUESTED** [review]
Very nice!! Just wondering though. Does it also auto-play on laptop version (similar to on mobile?

File: src/components/supportus/PhaseBudget.tsx (hunk 2 above)
Comment (CarsonKiibi) [reply src/components/supportus/PhaseBudget.tsx]: The figma didnt have a button for that on desktop, would be easy to add but might clutter desktop view

**Review by daniel-panhead — APPROVED** [review]
LGTM! pending @IcedTea2K

**Review by IcedTea2K — APPROVED** [review]

**Pull request merged**